contracts/root/RootERC20Predicate.sol

```diff
@@ -49,6 +52,7 @@ contract RootERC20Predicate is Initializable, IRootERC20Predicate {
             rootTokenToChildToken[nativeTokenRootAddress] = 0x0000000000000000000000000000000000001010;
             emit TokenMapped(nativeTokenRootAddress, 0x0000000000000000000000000000000000001010);
         }
+        _mapNative();
```

Comment:
@gretzke I'm concerned that mapping in the initializer won't correctly execute on the childchain due to the way genesis handling is done on polygon-edge atm. Any thoughts on this?

Reply:
Looks like mapping in the initializer didn't work

Reply:
Tested and it is not an issue once https://github.com/0xPolygon/polygon-edge/pull/1742 is merged